docs/developer-guide/developing.md

```diff
@@ -120,6 +120,35 @@ devServer: {
 
 * You have to run npm start to run mapstore client on port 8081, that is now connected to the local test back-end
 
+### Using local backend and setting up tomcat server
+* Download a tomcat standalone [here](https://tomcat.apache.org/download-90.cgi) and extract to a folder of your choice
```

Comment:
We point to Tomcat 7 here: https://mapstore.readthedocs.io/en/latest/developer-guide/requirements/

Maybe it's better to be consistent and always suggest the same version, or just link the above page for tomcat installation and deploy.

Reply:
Yes I am making the changes.

Reply:
@mbarto made the changes